tooling/modified-packages.js

```diff
@@ -38,11 +38,11 @@ const fileToPackage = (d) => {
  * @param {boolean=} options.dependents if true, also includes dependents of
  * updated packages
  * @param {boolean=} options.npm if true, compares to the last tag published to
- * npm instead of upstream/master
+ * npm instead of upstream/beta
```

Comment:
What is happening here? Should we change this to `next`?

Reply:
I think this isn't effective in the next branch. It was last updated in `next` branch 10 months ago and it still says `upstream/master`

So, I believe it is not relevant anymore

CC - @InteractiveTimmy